zulipterminal/config/keys.py

```diff
@@ -415,3 +435,9 @@ def commands_for_random_tips() -> List[KeyBinding]:
         for key_binding in KEY_BINDINGS.values()
         if not key_binding.get("excluded_from_random_tips", False)
     ]
+
+
+# Refer urwid/command_map.py
+for zt_cmd, urwid_cmd in ZT_TO_URWID_CMD_MAPPING.items():
```

Comment:
This specifically adds to the existing mapping, so it'd be useful to note this in a comment.

Reply:
@neiljp Do you mean in `ui_mappings.py`?